docs/user_manual/guides/sign_off_case.md

```diff
@@ -0,0 +1,38 @@
+# Complete Case Sign-Offs
+
+The final three QC steps are completed at the case level. These sign-offs are recorded in another
+system (Nabu), but can be completed from Dimsum.
+
+| QC Step | Description | Requirements |
+| ------- | ----------- | ------------ |
+| Analysis Review | Confirmation that the analysis data is good | Approval for each deliverable type - clinical report and/or data release |
+| Release Approval | Approval to release delierables to the collaborator | Approval for each deliverable type - clinical report and/or data release |
+| Release | Confirmation that a specific deliverable was delivered | Approval for each specific deliverable |
+
+When a case is stopped, release approval and release QC steps are still required in order to
+indicate that any partial results or failed deliverable have been delivered as required.
+
+1. Find and select the cases you wish to sign off
+    1. Cases can be found on the main QC Dashboard, or on the Cases table on the Project, Donor,
+       Case, or Requisition Details pages
+    2. Use the sorting and filter functions if necessary to help find the cases
+    3. Select the cases using the checkboxes at the left side of the table
+    4. You can use the checkbox in the header to select all cases currently shown in the table. If
+       there are multiple pages of cases in the table, this only selects the current page
+    5. Note that all the options you choose will apply to every case that you select. If you wish to
+       set a different status or note for some cases, you will need to select and sign them off them
+       separately
+2. Click the <img src="../../../images/signoff_button.png" alt="Sign Off button" width="64"> button at the top or bottom of the table
```

Comment:
is 'width' in pixels by default? 64 might be very small on eg a Retina display?

Reply:
The width attribute uses the same unit as `px` in CSS, which is (confusingly) not actually measure of pixels. https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units